app/assets/stylesheets/homepage.scss

```diff
@@ -0,0 +1,44 @@
+$coral: #ec4339;
+$white: #ffffff;
+
+.title-box {
+  background-color: $coral;
+  font-family: 'Alegreya Sans SC', sans-serif;
+  color: $white;
+  padding: 10px;
+  font-weight: bold;
+  letter-spacing: 1.25px;
+  font-size: 60px;
+  font-size: 5vw;
+ }
+
+.vid-image {
+  width: -webkit-fit-content;
```

Comment:
Properties should be ordered background-position-x, height, height, position, width, width